ingestion/src/metadata/ingestion/source/dashboard/mode/client.py

```diff
@@ -75,7 +75,7 @@ def fetch_all_reports(self, workspace_name: str) -> Optional[list]:
             dict
         """
         all_reports = []
-        response_collections = self.client.get(f"/{workspace_name}/{COLLECTIONS}")
+        response_collections = self.client.get(f"/{workspace_name}/{COLLECTIONS}?filter=all")
```

Comment:
filter=all param is added to get records from collections API

Reply:
@dixon2678 this looks good to me does this need any special permissions?

Reply:
@ulixius9 I have added my reply on the latest change 🙇‍♂️